Dockerfile

```diff
@@ -83,7 +90,8 @@ RUN <<EOF
 EOF
 
 COPY --from=build /built_app /built_app
-COPY scripts /built_app/scripts
+# Copy from test to force a dependency and ensure that the tests run
```

Comment:
😂 

Would it make sense to have the `test` image create some sort of "yeah these tests were passed" manifest to copy in here?

Reply:
Hmm. It seems like `meteortesting:mocha` can [generate an xunit report](https://github.com/Meteor-Community-Packages/meteor-mocha#write-tests-to-a-file)?